### ISSUE TITLE
Runaway process when calling .to_xml on Oga::XML::Text

### ISSUE THREAD

**Issue by bryanp** [YorickPeterse/oga#161]
The example below works in 2.3 but not 2.4 or 2.5.

Here's some HTML that reproduces the problem:

```html
<!DOCTYPE html>
<html>
<head>
  <title>
    foo
  </title>
</head>
</html>
```

This code never completes execution:

```ruby
require "oga"
doc = Oga.parse_html(File.read("doc.html"))
doc.children[0].to_xml
```

But this works fine:

```ruby
Oga::XML::Text.new(text: "\n").to_xml
```

Let me know if I can provide more info to track down the issue.

**Comment by YorickPeterse** [comment]
Fix for this is available in Oga 2.6.

**Comment by bryanp** [comment]
Awesome, thanks! ❤️
